crates/ra_project_model/src/cargo_workspace.rs

```diff
@@ -249,6 +251,18 @@ impl CargoWorkspace {
     pub fn workspace_root(&self) -> &Path {
         &self.workspace_root
     }
+
+    pub fn name_for_package(&self, package: &PackageData) -> String {
```

Comment:
```suggestion
    pub fn package_flag(&self, package: Package -> String {
```

package_name is overly generic for what this function actually does